homework4/task3.js

```diff
@@ -0,0 +1,8 @@
+function arrayDivisionFactory(num){
+    function divideF(arr){
+        let result = arr.map(el => el/num)
+        console.log(result)
+    }
+   divideF([10, 20, 25])
```

Comment:
INCORRECT
The same here
The function has a static array [10, 20, 25]
Expected result: It should be a possibility to set different arrays for function
You have example in task:
const divide5 = arrayDivisionFactory(5);
console.log(divide5([10, 20, 25])) // [2, 4, 5]
const divide3 = arrayDivisionFactory(3);
console.log(divide3([6, 21, 27])) // [2, 7, 9]